docs/pages/project/contributing/contributing-docs.md

```diff
@@ -170,11 +171,12 @@ Meshery documentation is made of these components:
 Most popular clipboard plugins like Clipboard JS require the manual creation of a new ID for each code snippet. A different approach is used here. For code snippets, we either use html tags or markdown in the following manner:
 
 ```
-   <pre><code>
+   <pre class="codeblock-pre"><div class="codeblock">
+   <code class="clipboardjs">
      code snippet
-   </code></pre>
+   </code></div></pre>
 ``` 
-_\<pre\> tags are optional unless the code snippet is in a paragraph format_
+_\<pre\> tags are optional unless the code snippet is in a paragraph format and also gives a terminal like effect to the code_
```

Comment:
```suggestion
_`\<pre\>` tags are optional unless the code snippet is in a paragraph format and also gives a terminal like effect to the code_
```

Reply:
@warunicorn19 or you can also make the `<pre>` tag bold, bc it  makes it look like a normal sentence unless the tag is highlighted 👀

Reply:
Should I just commit this change that youve suggested?😅

Reply:
I'm not exactly sure if it would fix it or not, as I didn't confirm it. But you can go ahead and try that though

Reply:
So, there was a problem with the front slash in markdown. So, instead added both the opening and closing <pre> tags, hope thats fine. Made the other change too.

Reply:
@warunicorn19 if someone has suggested changes you can commit them actually, it'll show as co-authored. And be careful on Signing off though 😂 coz GitHub will neglect it

Reply:
Ohh😅, actually that commit suggestion button has been tempting me for quite some time but I've never really used it😂.